packages/office-ui-fabric-react/src/components/Button/Button.types.ts

```diff
@@ -21,9 +21,11 @@ export interface IButton {
   dismissMenu: () => void;
 
   /**
-   * If there is a menu associated with this button and it is visible, this will open the menu
+   * If there is a menu associated with this button and it is visible, this will open the menu.
+   * shouldFocusOnContainer and shouldFocusOnMount are overrides to the ContextualMenu menuProps properties
```

Comment:
NIT: Can you change the documentation of the new parameters to be `@param` instead?

Reply:
And maybe add that if not passed in the provided/default values will be used

Reply:
agreed - merge when this comment is updated!